### PR TITLE
Feature/feed

### LINKED ISSUE #3: As a user, I will see a homepage
As a user, when I visit the Roomies website, I will see a homepage.
The homepage will have:
- the logo
- a button to view existing roomies' profiles

### LINKED ISSUE #4: Homepage styling
Style the Homepage

### LINKED ISSUE #13: As a user, I can view my dashboard
As a user, when I am logged in, I can view a list of potential roomies.

### LINKED ISSUE #14: As a user, I can view a nav bar
As a user, when I am logged in, I will view a nav bar with a view favorites button.

### LINKED ISSUE #16: Nav bar styling
Complete styling for nav bar

### LINKED ISSUE #17: Dashboard styling
Complete styling for dashboard view

### PR DESCRIPTION
**Pull request by AndieDrew**
##### Description

Using fake data, we are now displaying user cards on the dashboard.

##### What's changed & where should the reviewer start?

Create a fake data file data.js. Create Context.js file that contains context, import it to App.js where we wrap all nodes in context.provider. Created a dashboard component that maps through all the data provided by context after importing context. Create a Cards component that takes user data as probs and destructures it into jsx that is rendered by the dashboard.

##### Relevant Screenshots/GIFs

Screen Shot 2021-07-10 at 5 46 56 PM

#### Type of change
What is it?
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring

### Checklist:
- [x] My code follows the guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

**Review by Thee-Dust — COMMENTED** [review]
LGTM!

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+  }, [])
+
   return (
-    <main>
-        <Navbar />
-      <Switch>
-        <Route exact path='/' component={Home} />
-      </Switch>
-    </main>
+    <Context.Provider value={users}>
```
Comment (michannstoner) [inline src/components/App/App.js:19]: Nice work getting Context working!

File: src/components/Card/Card.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+export default function Card({ user }) {
+const { email, name, location, gender, age } = user.attributes
```
Comment (michannstoner) [inline src/components/Card/Card.js:4]: Love the use of destructuring here

**Review by michannstoner — APPROVED** [review]
Great work getting this mock data rendering on the dashboard! Should be pretty straightforward to swap it out once we're able to consume the API.

**Pull request merged**